### DISCUSSION TITLE
wagmi-cli generated hooks + wagmi 1.0.6 - overrides{} handling?

### DISCUSSION THREAD

**Discussion by bjfresh** [wevm/wagmi#2432] [Q&A] [answered]
Not documented in cli docs, as far as I can tell. Trying to migrate an existing project with lots of wagmi-cli usage to wagmi 1.0

From the [Wagmi 1.0 Migration Guide](https://wagmi.sh/react/migration-guide#usepreparecontractwrite) section on usePrepareContractWrite:

_The overrides parameter has been removed in favor of top-level parameters:_

```
- overrides: {
-   from: '0xA0Cf798816D4b9b9866b5330EEa46a18382f251e',
-   value: BigNumber.from('69420'),
- },
+ account: '0xA0Cf798816D4b9b9866b5330EEa46a18382f251e',
+ value: 69420n
```

In trying to convert a hook wrapping usePrepareContractWrite, it doesn't appear that the override params are surfaced.

So hooks using overrides don't work, whether implemented as

```
    overrides: {
      from:'0x...',
    },
```
or
```
  from: '0x...'
```
or
```
  account: '0x...'
```

- wagmi-cli 1.0.2
- wagmi 1.0.6

**Comment by jxom** [comment, marked as answer]
Good catch. We should have the ability to set an account override. I have a PR at #2433
